nginx.conf

```diff
@@ -64,7 +48,7 @@ http {
         proxy_set_header X-Forwarded-Host $host;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
         proxy_set_header X-Script-Name "<%= path %>";
-
+        proxy_set_header Cache-Control "s-maxage=(600)";
```

Comment:
Hmm, so this isn't coming through?

It appears that there might need to be a bit more set based on some examples I'm seeing.

I also came across the [NGINX Caching Guide](https://www.nginx.com/blog/nginx-caching-guide/), which might be useful in figuring out what to set (it also links to more specific documentation as well).

It looks like there might be a bit more that needs to be configured?  [This DigitalOcean guide](https://www.digitalocean.com/community/tutorials/understanding-nginx-http-proxying-load-balancing-buffering-and-caching#configuring-proxy-caching-to-decrease-response-times) also has some info around proxy cache configuration.